### PR TITLE
Implements extra questions page UI adjustments.

### PR DESCRIPTION
**Pull request by gcutrini**
re: https://tipit.avaza.com/project/view/232086#!tab=task-pane&task=3172382

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -150,6 +150,7 @@
   "devDependencies": {
     "@babel/core": "^7.17.8",
     "devcert": "1.1.0",
+    "gatsby-plugin-root-import": "^2.0.9",
```
Comment (smarcet) [inline package.json:153]: this should be dev ?

File: src/components/summit-my-orders-tickets/store/actions/ticket-actions.js [hunk 2]
```diff
@@ -311,7 +319,8 @@ export const editOwnedTicket = ({
         if (context === 'ticket-list') {
             dispatch(getUserTickets({ page: ticketPage }));
         } else {
-            dispatch(getUserOrders({ page: orderPage }))
+            dispatch(getUserOrders({ page: orderPage }));
```
Comment (smarcet) [inline src/components/summit-my-orders-tickets/store/actions/ticket-actions.js:322]: chain this with then to avoid RC

File: package.json (hunk 1 above)
Comment (gcutrini) [reply package.json:153]: Made a regular dependency.

File: src/components/summit-my-orders-tickets/store/actions/ticket-actions.js (hunk 2 above)
Comment (gcutrini) [reply src/components/summit-my-orders-tickets/store/actions/ticket-actions.js]: chained!

**Review by smarcet — APPROVED** [review]
LGTM

**Pull request merged**